aten/src/ATen/test/basic.cpp

```diff
@@ -283,6 +283,14 @@ void TestNegativeDim(DeprecatedTypeProperties& type) {
   ASSERT_ANY_THROW(tensor.reshape({-5, -5}));
 }
 
+void TestView(DeprecatedTypeProperties& type) {
```

Comment:
nit: would be good to comment why you need this test; that this tests the *Tensor* view path, not the *Variable* view path, which has different semantics (you could link this PR for more details).